tests/src/test/scala/org/apache/spark/sql/rapids/metrics/source/MockTaskContext.scala

```diff
@@ -66,7 +66,7 @@ class MockTaskContext(taskAttemptId: Long, partitionId: Int) extends TaskContext
 
   override private[spark] def killTaskIfInterrupted(): Unit = {}
 
-  override private[spark] def getKillReason() = None
+  override def getKillReason() = None
```

Comment:
I don't think this change was part of the original commit, was it?

Reply:
Yes, this is to fix DBX build.

Reply:
It seems going to be fixed in https://github.com/NVIDIA/spark-rapids/pull/10070 Will update accordingly once merged.